### PR TITLE
add register_repos(), more comments in __init__.py

### PR DESCRIPTION
**Pull request by FjorgeTheHuman**

### CODE REVIEW

**Comment by caughtquick** [comment]
Returning an empty list is probably a better idea then returning nothing

**Review by thewarsawpakt — COMMENTED** [review]
Looks good.

**Review by thewarsawpakt — APPROVED** [review]
Looks good.

**Review by caughtquick — APPROVED** [review]
Looks good

File: libtwirl/__init__.py [hunk 1]
```diff
@@ -32,5 +37,10 @@ def search_repos(pkgname, find_provides = True): # Function to search the synced
 		# Return the list of found packages
 		return packages
 	else:
-		return None
+		# Return None if there were no packages found
+		return None # Would it be preferred if this only returned an empty list instead of None?
```
[the remark this answers is not in the document]
Comment (FjorgeTheHuman) [reply libtwirl/__init__.py:41]: Did you mean `None` instead of one?

[the remark this answers is not in the document]
Comment (thewarsawpakt) [reply libtwirl/__init__.py:41]: yes, fixed.

[the remark this answers is not in the document]
Comment (FjorgeTheHuman) [reply libtwirl/__init__.py:41]: Ok. I'll be removing that comment next time I make a PR.

**Pull request merged**